src/components/Consent.jsx

```diff
@@ -237,133 +150,10 @@ export function Consent() {
               <ConsentTextEn />
             </div>
 
-            <Typography>
-              By clicking &ldquo;Next&ldquo;, you agree to participate. Before
-              we begin, please enter your:
+            <Typography paragraph>
+              By clicking &ldquo;Next&ldquo;, you agree to participate.
             </Typography>
           </Grid>
-          <Grid item xs={12}>
-            <FormControl
```

Comment:
I think it would be good to keep some type of user input (required checkbox) here to make sure that they agree that they have read the information and consent to participating in the study. Let me find an example of a survey online that does this since it seems like standard practice.

Reply:
![image](https://user-images.githubusercontent.com/7852732/178306992-7432955b-3b98-4c65-968d-d134e64711ac.png)
![image](https://user-images.githubusercontent.com/7852732/178307108-1c30c0fb-9abb-4fce-a172-6f7bd0d83b3f.png)